File: conformance/tests/if-without-else/if-without-else.runar.ts
import { SmartContract, assert } from 'runar-lang';

class IfWithoutElse extends SmartContract {
  readonly threshold: bigint;

  constructor(threshold: bigint) {
    super(threshold);
    this.threshold = threshold;
  }

  public check(a: bigint, b: bigint): void {
    let count: bigint = 0n;
    if (a > this.threshold) { count = count + 1n; }
    if (b > this.threshold) { count = count + 1n; }
    assert(count > 0n);
  }
}

File: conformance/tests/if-without-else/if-without-else.runar.py
from runar import SmartContract, Bigint, public, assert_

class IfWithoutElse(SmartContract):
    threshold: Bigint

    def __init__(self, threshold: Bigint):
        super().__init__(threshold)
        self.threshold = threshold

    @public
    def check(self, a: Bigint, b: Bigint):
        count: Bigint = 0
        if a > self.threshold:
            count = count + 1
        if b > self.threshold:
            count = count + 1
        assert_(count > 0)
